web-console/src/lib/compositions/streaming/import/useDefaultRows.ts

```diff
@@ -9,7 +9,7 @@ import { match } from 'ts-pattern'
 
 import { GridApi } from '@mui/x-data-grid-pro'
 
-export const getDefaultValue = (columntype: ColumnType) =>
+export const getDefaultValue = (columntype: ColumnType): SQLValueJS =>
```

Comment:
what does this mean?

Reply:
`: SQLValueJS` states the return type; with it the on-hover function definition tooltip resolves the return type as a concise `: SQLValueJS`, without - an equivalent, but less convenient `string | number | boolean | BigNumber | dayjs.Dayjs`